pandas/tseries/tests/test_tslib.py

```diff
@@ -589,80 +589,100 @@ def test_parsers_quarter_invalid(self):
             self.assertRaises(ValueError, tools.parse_time_string, case)
 
     def test_parsers_dayfirst_yearfirst(self):
-        raise nose.SkipTest("skipping until comprehensive fixes for dateutil, "
-                            "xref #12944")
+        tm._skip_if_no_dateutil()
+
+        # OK
+        # 2.5.1 10-11-12   [dayfirst=0, yearfirst=0] -> 2012-10-11 00:00:00
+        # 2.5.2 10-11-12   [dayfirst=0, yearfirst=1] -> 2012-10-11 00:00:00
```

Comment:
not sure what you mean, this is just printing out the 4 combinations of booleans (True/False).